demo/api/schema.gql

```diff
@@ -247,6 +248,7 @@ type DamFile {
   title: String
   altText: String
   archived: Boolean!
+  copyOf: String
```

Comment:
in the graphql schema I expect for copyOf a nested object. If you really need the id, it could be neamed copyOfFileId or so

Reply:
You are right. I converted copyOf to be a File. Also I removed it from GraphQL since we don't plan to use it in the admin at the moment.